taskcluster/ci/nimbledroid/kind.yml

```diff
@@ -12,30 +12,37 @@ kind-dependencies:
     - signing
 
 
+job-defaults:
+    description: Upload APKs to Nimbledroid for performance measurement and tracking
+    run:
+        commands:
+            - [wget, {artifact-reference: '<signing/public/build/armeabi-v7a/geckoNightly/target.apk>'}, '-O', target.apk]
+            - [python, automation/taskcluster/upload_apk_nimbledroid.py, target.apk, .nimbledroid_token]
+        secrets:
+            - name: project/mobile/fenix/nimbledroid
+              key: api_key
+              path: .nimbledroid_token
+    run-on-tasks-for: []
+    treeherder:
+        kind: test
+        platform: android-all/opt
+        symbol: nimbledroid
```

Comment:
Nit: Remove this line and add 

```yaml
            treeherder:
                symbol: nightly(nimbledroid)
```

to the nightly section. The treeherder symbol must be unique across the jobs.